### PR TITLE
Introduce navigation menu

### PR DESCRIPTION
**Pull request by dbl7**
add colors file
add normolize.css

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -18,6 +19,7 @@ import { GroceryModule } from 'src/app/grocery/grocery.module';
   ],
   declarations: [
     AppComponent,
+    NavigationComponent,
```
Comment (dgutoryan) [inline src/app/app.module.ts]: It should be better to create core module or app layout module, or smth like this and move navigation component there

File: src/app/navigation/navigation.component.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
+@import "src/assets/styles/colors.scss";
```
Comment (dgutoryan) [inline src/app/navigation/navigation.component.scss]: Add style preprocessor option to `angular.json` for easier import

Comment (dgutoryan) [inline src/app/navigation/navigation.component.scss]: Add style preprocessor option to `angular.json` for easier import

File: src/app/app.module.ts (hunk 1 above)
Comment (dgutoryan) [inline src/app/app.module.ts]: It should be better to create core module or app layout module, or smth like this and move navigation component there

File: src/app/navigation/navigation.component.scss (hunk 2 above)
Comment (dbl7) [reply src/app/navigation/navigation.component.scss]: yeah, great 👍

File: src/app/app.module.ts (hunk 1 above)
Comment (dbl7) [reply src/app/app.module.ts]: Not quite, navigation isn't a part of the core or layout module. It's a part of the whole application and it's better to keep it in the application module.

**Pull request merged**